### ISSUE TITLE
Setting screenshooter rootDir does not create full path of folders/subfolders

### ISSUE THREAD

**Issue by FibreFoX** [arquillian/arquillian-recorder#65]
##### Issue Overview

While using a full arquillian-stack (arquillian-drone, arquillian-graphene, arquillian-graphene-recorder, arquillian-drone-browserstack-extension) and running with JUnit, I came across some problem.  
My project contains a configuration of the `maven-surefire-plugin` with multiple executions, where all the browser-versions and operating-systems are provided via `<systemPropertyVariables>`. For better organization, I tried to put all generated screenshots inside subfolders, but running `mvn clean test` broke, because `target/screenshots` did not exist, while I configured it to be `target/screenshots/browserAndOperatingSystemSubfolder`.

**TL;DR**
rootDir does not create full path of folders, one below existing folders

##### Expected Behaviour
All folders are created and screenshots are generated.

##### Current Behaviour

It fails to run my tests.

##### Steps To Reproduce
1. create test-project
2. [step 2]

##### Additional Information
Current workaround by creating the folder via ant-plugin ;)
```xml
<plugin>
    <artifactId>maven-antrun-plugin</artifactId>
        <executions>
        <execution>
            <id>create-screenshots-folder</id>
            <phase>generate-sources</phase>
            <goals>
                <goal>run</goal>
            </goals>
            <configuration>
                <tasks>
                    <mkdir dir="target/screenshots"/>
                </tasks>
            </configuration>
        </execution>
    </executions>
</plugin>
```

My maven dependencies/imports:
```

<properties>
    <version.arquillian-universe>1.0.0.Alpha6</version.arquillian-universe>
</properties>

<dependencyManagement>
    <dependencies>
        <dependency>
            <groupId>org.arquillian</groupId>
            <artifactId>arquillian-universe</artifactId>
            <version>${version.arquillian-universe}</version>
            <scope>import</scope>
            <type>pom</type>
        </dependency>
    </dependencies>
</dependencyManagement>

<dependencies>
    <dependency>
        <groupId>junit</groupId>
        <artifactId>junit</artifactId>
        <version>4.12</version>
        <scope>test</scope>
    </dependency>
    <dependency>
        <groupId>org.arquillian.universe</groupId>
        <artifactId>arquillian-junit</artifactId>
        <scope>test</scope>
        <type>pom</type>
    </dependency>
    <!-- arquillian-graphene will get arquillian-drone -->
    <!-- arquillian-graphene-recorder will get arquillian-graphene -->
    <dependency>
        <groupId>org.arquillian.universe</groupId>
        <artifactId>arquillian-graphene-recorder</artifactId>
        <scope>test</scope>
        <type>pom</type>
    </dependency>
    
    <dependency>
        <groupId>org.jboss.arquillian.extension</groupId>
        <artifactId>arquillian-drone-browserstack-extension</artifactId>
        <scope>test</scope>
        <version>2.1.0.Alpha1</version>
    </dependency> 

</dependencies>
```

**`mvn -v`**
```
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T17:41:47+01:00)
Maven home: C:\develop\tools\maven\apache-maven-3.3.9\bin\..
Java version: 1.8.0_121, vendor: Oracle Corporation
Java home: c:\Program Files\Java\jdk1.8.0_121\jre
Default locale: de_DE, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "dos"
```

**Comment by smiklosovic** [comment]
Thank you very much for reporting the issue. I believe the change you request is not hard to implement and you are very welcome to do so. I case you hesitate to do it, I believe guys like @MatousJobanek would be glad to help

**Comment by FibreFoX** [comment]
Looks like this counts for the video-recorder too:
https://github.com/arquillian/arquillian-recorder/search?utf8=%E2%9C%93&q=rootDir

**Comment by FibreFoX** [comment]
@smiklosovic pull-request created ;) but affects reporter-api, video-api and screenshot-api
